repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh

```diff
@@ -196,8 +196,8 @@ ibdmp() {
 
 bring_up_network() {
     if [ -f /etc/leapp-initram-network-manager ]; then
-	# NOTE(ivasilev) Reverting the change to see if it caused the crash
-	. /lib/dracut/hooks/cmdline/99-nm-config.sh
+    # NOTE(ivasilev) Reverting the change to see if it caused the crash
+    . /lib/dracut/hooks/cmdline/99-nm-config.sh
         . /lib/dracut/hooks/initqueue/settled/99-nm-run.sh
```

Comment:
```suggestion
            . /lib/dracut/hooks/initqueue/settled/99-nm-run.sh
```

Reply:
lol :D how I chieved this wrong replacement in such a simple change :-D *facepalm* :dagger:  :jp: